packages/atlas/src/components/StudioEntrypoint/StudioEntrypoint.tsx

```diff
@@ -35,3 +35,5 @@ export const StudioEntrypoint: FC<StudioEntrypointProps> = ({ enterLocation }) =
 
   return <StudioLoading />
 }
+
+export default StudioEntrypoint
```

Comment:
we should't change it in this PR, but generally we should avoid default export in files

Reply:
`React.lazy` requires default export